actions/get_dependencies/get_dependencies.sh

```diff
@@ -156,6 +156,26 @@ get_lapacke() {
   ${APT_COMMAND} update
   ${APT_GET_COMMAND} install liblapacke liblapacke-dev
 }
+# Wraps installing libfort
+#
+# Usage:
+#   get_libfort
+get_libfort() {
+  if [ -z "${CACHE_LIBFORT}" ]; then
+    export INSTALL_PATH=`pwd`/install
+    wget --no-check-certificate --content-disposition https://codeload.github.com/seleznevae/libfort/tar.gz/refs/tags/v${libfort_version}
+    tar -zxf libfort-${libfort_version}.tar.gz
+    cd libfort-${libfort_version}
+    export CXX=`which g++`
+    export CC=`which gcc`
+    ../cmake-${cmake_version}-${arch}/bin/cmake -GNinja -H. -Bbuild -DCMAKE_INSTALL_PREFIX=${INSTALL_PATH}
+    cd build
+    ninja
+    ninja install
```

Comment:
CMake has a make system agnostic way to build and install a project after configuration:
```suggestion
../cmake-${cmake_version}-${arch}/bin/cmake --build build --target install
```